server/status.go

```diff
@@ -155,6 +165,65 @@ func (s *statusServer) handleLocalStatus(w http.ResponseWriter, r *http.Request,
 	w.Write(b)
 }
 
+// handleLocalLogs handles GET requests for list of available logs.
+func (s *statusServer) handleLocalLogs(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	log.Flush()
+	logFiles, err := log.ListLogFiles()
+	if err != nil {
+		log.Error(err)
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	b, contentType, err := util.MarshalResponse(r, logFiles, []util.EncodingType{util.JSONEncoding})
```

Comment:
The vulnerability is a bypass of the same-origin policy: any website you visit in a vulnerable browser could read any json URLs you have access to iff those urls return top-level arrays. GET versus POST makes no difference; we can and should continue to allow GET for these status urls.